spec/controllers/staff_users_controller_spec.rb

```diff
@@ -28,29 +28,66 @@ def authenticate_staff_user(user)
   end
 
   context "New Account Authentication" do
-    it 'creates a staff user when provided valid username and password' do
-      user = build :staff_user
+    def create_staff_user(username = "newuser12345", cultural_center = "MoMA")
```

Comment:
I would probably require the `username` and `cultural_center` arguments here, because I think it would make the logic you're testing more obvious.